enterprise/models.py

```diff
@@ -3103,10 +3112,30 @@ def is_valid(self):
         is_not_expired = now < self.expiration_date
         usage_count = self.linked_enterprise_customer_users.count()
         is_usage_under_limit = usage_count < self.usage_limit
-        return is_not_expired and is_usage_under_limit
+        return is_not_expired and is_usage_under_limit and self.is_active
```

Comment:
nit: I think this reads a little more naturally if `self.is_active` is the first clause of this conjunction.  And it might save a query, too.